test/request-filtering-agent.test.ts

```diff
@@ -137,11 +138,37 @@ describe("request-filtering-agent", function() {
             }
         }
     });
-    it("should not request because Socket is closed", async () => {
+    it("IPv4: should not request because Socket is closed", async () => {
         const privateIPs = [
             `http://0.0.0.0:${TEST_PORT}`, // 0.0.0.0 is special
             `http://127.0.0.1:${TEST_PORT}`, //
-            `http://A.com@127.0.0.1:${TEST_PORT}` //
+            `http://A.com@127.0.0.1:${TEST_PORT}`
+        ];
+        for (const ipAddress of privateIPs) {
+            try {
+                await fetch(ipAddress, {
+                    agent: useAgent(ipAddress),
+                    timeout: 2000
+                });
+                throw new ReferenceError("SHOULD NOT BE CALLED");
+            } catch (error) {
+                if (error instanceof ReferenceError) {
+                    assert.fail(error);
+                }
+                assert.ok(/Socket is closed/i.test(error.message), `Failed at ${ipAddress}, error: ${error}`);
+            }
+        }
+    });
+    // TODO: Travis CI does not support IPv6
+    // https://docs.travis-ci.com/user/reference/overview/
+    // https://github.com/travis-ci/travis-ci/issues/8891
+    (IS_TRAVIS ? it.skip : it)("IPv6: should not request because Socket is closed", async () => {
```

Comment:
Travis CI does not support IPv6...

We need to other CI servier that support IPv6 network...
In locally, These tests are passed.